bigbluebutton-html5/imports/api/group-chat-msg/server/publishers.js

```diff
@@ -15,17 +14,7 @@ function groupChatMsg(credentials) {
 
   Logger.info(`Publishing group-chat-msg for ${meetingId} ${requesterUserId} ${requesterToken}`);
 
-  return GroupChat.find({
-    $or: [
-      {
-        access: CHAT_ACCESS_PUBLIC,
-        meetingId,
-      }, {
-        users: { $in: [requesterUserId] },
-        meetingId,
-      },
-    ],
-  });
+  return GroupChatMsg.find({ meetingId });
```

Comment:
Won't this result in the publishing of every chat message in the meeting? Seems like a security hole.

Reply:
Yeah it's a security hole, i'm fixing it.

Reply:
Seems we still expose all group chat to anyone subscribed